src/backend/distributed/sql/downgrades/citus--12.0-1--11.3-1.sql

```diff
@@ -43,4 +43,7 @@ DROP FUNCTION pg_catalog.citus_internal_unregister_tenant_schema_globally(Oid, t
 #include "../udfs/citus_tables/11.1-1.sql"
 #include "../udfs/citus_shards/11.1-1.sql"
 
+DROP VIEW IF EXISTS public.citus_schemas;
+DROP VIEW IF EXISTS pg_catalog.citus_schemas;
```

Comment:
should we add a comment to tell that we use IF EXISTS syntax because we might create the view in either of those schemas conditionally